webpack/manifest.ts

```diff
@@ -101,7 +102,7 @@ const AUTHOR_URL = 'https://madfish.solutions';
 
 const PERMISSIONS = ['storage', 'unlimitedStorage', 'clipboardWrite', 'activeTab'];
 
-const HOST_PERMISSIONS: string[] = ['http://localhost:8732/'];
+const HOST_PERMISSIONS: string[] = ['http://localhost:8732/', 'http://localhost:3000/'];
```

Comment:
is this necessarily? i'm not sure but our extension might get disabled because of this host permissions change. pls see: https://groups.google.com/a/chromium.org/g/chromium-extensions/c/EnUmtHWOI9o (mb not, and I misunderstood something, but when i changed the permission array extension got disabled, so this needs to be double checked).